emails/templates/emails/first_forwarded_email.html

```diff
@@ -0,0 +1,573 @@
+    }
+
+    #what-you-can-do-with-relay p {
+      font-family: inter, system-ui, sans-serif;
+      line-height: 140%;
+      text-align: left;
+      word-wrap: break-word;
+    }
+
+    #what-you-can-do-with-relay p:first-child {
+      margin-bottom: 8px;
+    }
+
+    #what-you-can-do-with-relay:before {
+      content: '';
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      /* sets order for clipped background */
+      z-index: -1;
+      /* set to the same width of the border for clipping */
+      margin: -3px;
+      border-radius: inherit;
+      background: radial-gradient(100% 100% at 100% 0%, #9D62FC 0%, #FD3296 100%);
+    }
+
+    #relay-email-header {
+      padding: 12px 25px;
+    }
+
+    #relay-email-header,
+    #relay-email-footer {
+      font-family: 'inter', Arial, sans-serif;
+      color: #FFFFFF;
+      font-size: 12px;
+    }
+
+    a.container-link {
+      transition: all 0.2s ease;
+      text-underline-offset: 2px;
+      font-family: 'inter medium', Arial, sans-serif;
+    }
+
+    a.container-link:hover {
+      color: #20123ad5;
+      transition: all 0.2s ease;
+    }
+
+    a.container-link {
+      color: #FFFFFF;
+      font-size: 12px;
+    }
+
+    a.container-link:hover,
+    a.container-link:focus,
+    a.container-link:active {
+      transition: all 0.2s ease;
+    }
+
+    .email-trackers-removed-icon {
+      margin-right: 5px;
+      vertical-align: bottom;
+    }
+
+    /* this deals with long email addresses */
+    .forwarded-from-email {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        max-width: 350px;
+    }
+
+    @media screen and (max-width: 768px) {
+      .forwarded-from-email {
+        width: 250px;
+      }
+
+      .footer-block {
+        display: block;
+        width: 100%;
+        text-align: center;
+      }
+
+      .footer-block .container-link img {
+        height: 45px;
+        width: auto;
+      }
+
+      #relay-email-header {
+        padding: 8px 4px;
+      }
+      
+      #hero td {
+        display: block;
+        padding: 0 0 0 24px !important;
+      }
+      .arrow-img {
+        display: none !important;
+      }
+      #hero-mask-img {
+        text-align: center;
+      }
+      #hero-desc {
+        max-width: 100% !important;
+        margin-bottom: 32px;
+      }
+      
+    }
+
+    @media screen and (max-width: 490px) {
+      #our-promise-section *:not(:last-child) {
+        margin-bottom: 32px;
+      }
+      #what-you-can-do-with-relay {
+        padding: 16px;
+      }
+      #what-you-can-do-with-relay td img {
+        min-width: 20px;
+      }
+      #view-your-dashboard {
+        margin: 32px 0 !important;
+      }
+      .forwarded-from-email {
+        width: 200px;
+      }
+      .header-block-left *,
+      .header-block-right * {
+        font-size: 8px !important;
+      }
+      .email-trackers-removed-icon {

```

Comment:
question (non-blocking): In terms of accessibility, we should keep in mind that some font-sizes are too small, especially on mobile. The minimum font size to use on mobile or the web can vary depending on the context, but a common recommendation, IIRC, is around 16 pixels (px) for body text. This size ensures readability for most users without needing to zoom in. We can always check the size of that using dev tools - the values under the computed tab. 

For accessibility reasons, a minimum font size is crucial because it ensures that text is legible for all users, including those with visual impairments. I have bad eyesight and I had to squint to see the text on mobile in the header. Small text can be difficult to read for people with low vision or those using screen readers. By using a minimum font size, you make your content more inclusive and accessible to a wider audience.

As an added bonus, web accessibility guidelines, such as WCAG (Web Content Accessibility Guidelines), recommend a minimum contrast ratio between text and its background. A minimum font size helps in meeting these contrast requirements, making text more discernible and improving the overall accessibility of anything we work on. 

@Vinnl I don't know if you have any additional input here.